ugvc/mrd/featuremap_utils.py

```diff
@@ -305,6 +341,12 @@ def process_records(self, records: list[pysam.VariantRecord]) -> list[pysam.Vari
                 record.info[self.NEXT_N_BP] = ref_around_snv[
                     central_base_ind + 1 : central_base_ind + self.motif_length_to_annotate + 1
                 ]
+                record.info[self.NEXT_1] = ref_around_snv[central_base_ind + 1]
```

Comment:
I think this is better be done in a loop.
can save the array of 6 feature names.

Reply:
done